scripts/build

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
Any reason not to rename this file to `cibuild`? It seems like the assembly JAR is what we'd want out of a `cibuild` process (in addition to the execution of some tests).